v-next/hardhat/package.json

```diff
@@ -85,7 +85,7 @@
     "typescript": "~5.8.0"
   },
   "dependencies": {
-    "@ignored/edr-optimism": "0.13.0-alpha.7",
```

Comment:
This was a last minute addition to the Alpha that I am reverting as it breaks one of the default scripts when running under optimism.